WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -236,3 +237,50 @@ $(function() {
 Math.trunc = Math.trunc || function(x) {
   return x < 0 ? Math.ceil(x) : Math.floor(x);
 };
+
+// Setting up bootstrap-table
+$(function() {
+  // Hide loading box
+  $('table[data-toggle="table"]').bootstrapTable('hideLoading');
+
+  // It's not necessary when bootstrap-table will be distributed with this merged:
+  // https://github.com/wenzhixin/bootstrap-table/pull/2145
+  // (and the appropriate gem will be updated)
+  // -------------------------------------------------------------------
+  // Triggered when a sort arrow is clicked but before a table is sorted
+  $('table').on('sort.bs.table', function(e, name, order) {
```

Comment:
That doesn't work, I think that the event is triggered by whole table instead of a thead.